app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -5,8 +5,14 @@ class MS.Views.TimelineTableView extends Backbone.View
 
   initialize: (options) ->
     new MS.Views.AutoLoadingIndicatorView()
-    @loanId = options.loanId
-    @groupModal = new MS.Views.ProjectGroupModalView(loanId: @loanId, success: @refresh.bind(@))
+    @projectId = options.projectId
+
+    if options.projectType == 'BasicProject'
```

Comment:
Written in this syntax because the below breaks, with or without the parentheses. The below returns `true` or `false` instead of the expected keys.
`@urlComponent = (options.projectType == 'BasicProject') ? 'basic-projects' : 'loans'`

The same syntax is used in another file.